fsi/body.go

```diff
@@ -81,3 +82,74 @@ func GetBody(dirPath string, format dataset.DataFormat, fcfg dataset.FormatConfi
 
 	return dsio.ConvertFile(file, structure, st, limit, offset, all)
 }
+
+// GetBodyAsInterface is an FSI version of base.ReadBodyAsInterface
+func GetBodyAsInterface(dirPath string, offset, limit int, all bool) (interface{}, error) {
```

Comment:
like the `base` versions, maybe this should be renamed to `GetBody` & the current `GetBody` should be `ReadBody` or `ReadBodyBytes`